File: 2556-ConvertTheTemperature/2556-ConvertTheTemperature.py
# Last updated: 9/3/2025, 10:46:36 AM
class Solution:
    def convertTemperature(self, celsius: float) -> List[float]:
         kelvin= celsius + 273.15;
         fahrenheit=celsius * 1.80 + 32.00;
         return [round(kelvin, 5), round(fahrenheit, 5)]
        

File: 2556-ConvertTheTemperature/2556-ConvertTheTemperature.java
// Last updated: 9/3/2025, 10:46:42 AM
class Solution {
    public double[] convertTemperature(double celsius) {
       double kelvin= celsius + 273.15;
       double  fahrenheit=celsius * 1.80 + 32.00;
       double [] result = {Math.round(kelvin * 100000.0) / 100000.0, Math.round(fahrenheit * 100000.0) / 100000.0};
      return result;

    
    }
}